### PR TITLE
feat(scheduler): adapt edas call chains to kubernetes

### PR DESCRIPTION
**Pull request by Muzry**
#### What type of this PR

Add one of the following kinds:
/kind feature

#### What this PR does / why we need it:
- feat(scheduler): adapt edas call chains to kubernetes
#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](paste your link here)

#### Specified Reviewers:

/assign @luobily  @sixther-dc 

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: modules/scheduler/executor/plugins/edas/edas.go [hunk 1]
```diff
@@ -2165,26 +2156,20 @@ func (e *EDAS) CapacityInfo() apistructs.CapacityInfoData {
...
+	}
+	r.ProdCPUOverCommit = 1.0
+	r.DevCPUOverCommit = 1.0
+	r.TestCPUOverCommit = 1.0
+	r.StagingCPUOverCommit = 1.0
+	r.ProdMEMOverCommit = 1.0
+	r.DevMEMOverCommit = 1.0
+	r.TestMEMOverCommit = 1.0
+	r.StagingMEMOverCommit = 1.0
+
```
Comment (sixther-dc) [inline modules/scheduler/executor/plugins/edas/edas.go]: This place is the same as before, edas does not need project quota

**Review by sixther-dc — COMMENTED** [review]
/lgtm

File: modules/scheduler/executor/plugins/edas/edas.go (hunk 1 above)
Comment (Muzry) [reply modules/scheduler/executor/plugins/edas/edas.go]: done

**Pull request merged**